src/pybind/mgr/dashboard/services/ceph_service.py

```diff
@@ -190,6 +209,70 @@ def get_rate(cls, svc_type, svc_name, path):
             return differentiate(*data[-2:])
         return 0.0
 
+    @classmethod
+    def get_client_perf(cls):
+        pools_stats = mgr.get('osd_pool_stats')['pool_stats']
+
+        client_perf = {
+            cls.POOL_STAT_READ_BYTES_PER_SEC: 0,
+            cls.POOL_STAT_READ_OP_PER_SEC: 0,
+            cls.POOL_STAT_RECOVERY_BYTES_PER_SEC: 0,
+            cls.POOL_STAT_WRITE_BYTES_PER_SEC: 0,
+            cls.POOL_STAT_WRITE_OP_PER_SEC: 0,
+        }
+
+        for pool_stats in pools_stats:
+            client_io = pool_stats[cls.POOL_STAT_CLIENT_IO_RATE]
+            if cls.POOL_STAT_READ_BYTES_PER_SEC in client_io:
+                client_perf[cls.POOL_STAT_READ_BYTES_PER_SEC] +=\
```

Comment:
Through the use of many long static variables this isn't as readable as you intended.

Reply:
IMHO it's a subjective matter.

Reply:
I would like to point out that I only found this usage in `rbd.py` and  `rbd_mirrowing.py` where the same object is used.

IMHO it's not a subjective matter if it is mostly avoided except for the only case where it's suitable.

Reply:
Replace those static variables with the actual strings before merging this PR please.

Reply:
The problem is that this string is only used once. If I have a typo in the object definition or in the call does not make a difference. As said it's only used once, but you will think it's used in many places instead, if you only see those four lines above.